android/trifle/src/main/java/app/cash/trifle/KeyHandle.kt

```diff
@@ -0,0 +1,57 @@
+package app.cash.trifle
+
+import android.security.keystore.KeyGenParameterSpec
+import android.security.keystore.KeyProperties
+import android.util.Log
+import java.security.KeyPairGenerator
+import java.security.KeyStore
+import java.security.spec.ECGenParameterSpec
+
+data class KeyHandle internal constructor(private val alias: String) {
```

Comment:
Does this being a data class make sense? The main point of being a data class is for immutable data structures and access to the `copy` function. If this stays a data class, a user could do something like:
```
val handle = KeyHandle.generateKeyHandle(alias)
val newHandle = handle.copy(alias = "newAlias")
```

Which feels a little odd if we want them to be using the `generateKeyHandle` function to be creating these.

Reply:
good point, removed!

Reply:
Removed.